contracts/core/SoundEditionV2_1.sol

```diff
@@ -778,7 +780,7 @@ contract SoundEditionV2_1 is ISoundEditionV2_1, ERC721AQueryableUpgradeable, ERC
      * @param roles A roles bitmap.
      */
     function _requireOnlyRolesOrOwner(uint256 roles) internal view {
-        address sender = LibMulticaller.sender();
+        address sender = LibMulticaller.senderOrSigner();
```

Comment:
Concerning the `MulticallerWithSigner`, how thorough does the review need to be? The library seems covered by the audit, so we're really just considering if we think the function usage here is appropriate. Is this correct?